qemu/tests/rng_hotplug.py

```diff
@@ -99,7 +105,14 @@ def stop_rngd(vm):
         for device in device_ids:
             unplug_rng(vm, device)
 
-    for i in xrange(repeat_times):
+    if sub_type_after_unplug:
+        error_context.context("Running sub test %s after unplug" % sub_type_after_unplug, logging.info)
+        utils_test.run_virt_sub_test(test, params, env, sub_type_after_unplug)
+
+    if sub_type_after_unplug == "shutdown":
+        return
```

Comment:
these two line is useless here, because line 110 is effected

Reply:
line 110 do reboot/shutdown as I expected.
line 112&113 is to skip followed testing if guest shutdown